z_infra_provisioning/cloud_infra_center/ocp_upi/README.md

```diff
@@ -113,11 +113,60 @@ Provision a Red Hat machine based on the following specifications, or using exis
 * Python3
 * Ansible
 * jq
+* wget
+* git
+* firewalld
 * Python modules required in the playbooks. Namely:
   * openstackclient 
   * openstacksdk >= 0.57.0
   * netaddr
 
+**Install:**
+
+Use the following command to register the system, then automatically associate any available subscription matching that system:
+```sh
+sudo subscription-manager register --username <username> --password <password> --auto-attach
+```
+After registration, use the following command to enable ansible repository.
+```sh
+sudo subscription-manager repos --enable=ansible-2.8-for-rhel-8-s390x-rpms 
```

Comment:
One more question, why we use low version ansible-2.8 here?

Reply:
I had not tested other version yet, maybe we could add one ansible upgrade version on 115.